### DISCUSSION TITLE
When the attack system will be implement? [BIG THANK ]

### DISCUSSION THREAD

**Discussion by dep1engenharia** [lanedirt/OGameX#281] [Q&A] [answered]
Hi lanedirt,

Thank you for your exceptional work on the OGameX clone. I’m truly impressed with the quality of the game. Looking forward to seeing more exciting developments in the future.

Also do you know when the attack option will be available?

Thank you

**Comment by dep1engenharia** [comment]
NVM... I still very impressed by how organized and efficient you’re running this project. The "Add attack mission and battle report logic" branch looks great. Keep up the excellent work!

**Comment by lanedirt** [comment, marked as answer]
Hi @dep1engenharia,

Thanks for the kind words, always nice to hear 😄 ! Yes as you have noticed work is already underway on the attack mission and battle logic. It's quite a big feature but I hope to have the basic workings finished and in the game somewhere in the coming 2 weeks.

**Reply by dep1engenharia** [reply]
Hi @lanedirt 

I can't believe you're doing all this work alone.

It's mind blowing. The game so far is great!! 

I hope you can continue the developments!! 

I will try to understand the framework to see if in the feature I can do some contributions to the project.

Until than.. MANY THANKS!!!!  🙏👌😁🍺🍺🍺💪
